pkg/skaffold/util/util.go

```diff
@@ -154,6 +154,76 @@ func ReadConfiguration(filename string) ([]byte, error) {
 	}
 }
 
+// A type holding configuration properties
+type ConfigProperties map[string]string
+
+// Tests if a file exists
+func FileExists(filapath string) bool {
+	if _, err := os.Stat(filapath); !os.IsNotExist(err) {
+		if err == nil {
+			return true
+		}
+	}
+	return false
+}
+
+// Reads an environment file  that contains a set of environment variables
```

Comment:
nit: extra space between `file` and `that`

Reply:
also, could you expand your comment a bit to detail what is returned by this function